examples/bridge-app/linux/bridged-actions-stub.cpp

```diff
@@ -131,7 +131,7 @@ CHIP_ERROR ActionsAttrAccess::Read(const ConcreteReadAttributePath & aPath, Attr
 }
 } // anonymous namespace
 
-void MatterActionsPluginServerInitCallback()
-{
-    AttributeAccessInterfaceRegistry::Instance().Register(&gAttrAccess);
-}
+// void MatterActionsPluginServerInitCallback()
```

Comment:
Why are all these being commented out?  If they are replaced by something that uses the new setup (where?), then they should just be removed.